tutorials/3d/fps_tutorial/part_five.rst

```diff
@@ -541,12 +541,12 @@ ______
 
 If ``grabbed_object`` is not ``null``, then we need to throw the :ref:`RigidBody <class_RigidBody>` the player is holding.
 
-We first set the :ref:`RigidBody <class_RigidBody>` we are holding mode to ``MODE_RIGID``.
+We first set the mode of the :ref:`RigidBody <class_RigidBody>` we are holding to ``MODE_RIGID``.
 
 .. note:: This is making a rather large assumption that all the rigid bodies will be using ``MODE_RIGID``. While that is the case for this tutorial series,
           that may not be the case in other projects.
 
-          If you have :ref:`RigidBody <class_RigidBody>`'s with different modes, you may need to store the mode of the :ref:`RigidBody <class_RigidBody>` you
+          If you have :ref:`RigidBody <class_RigidBody>`-ies with different modes, you may need to store the mode of the :ref:`RigidBody <class_RigidBody>` you
```

Comment:
"``RigidBody``-ies" looks awkward too. Best avoid the plural if it can't be composed here, and say "``RigidBody`` nodes".

Reply:
Or just "rigid bodies" as I used eventually, we don't need so many hyperlinks everywhere.